jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -246,10 +246,12 @@ properties:
             enum: [OK, SUSPECT]
             fits_keyword: ENG_QUAL
             blend_table: True
-          pointing_engdb_quality:
+          engdb_pointing_quality:
```

Comment:
Will this change break calls like https://github.com/spacetelescope/jwst/blob/20d5bbbaeb6c1f25fb23bf12550c01be10c881f7/jwst/lib/set_telescope_pointing.py#L467 ? I only found references to 'pointing_engdb_quality' in set_telescope_pointing, at least.

Reply:
D'oh! Yes, it sure would break those. Updating the script for the next commit. Good catch!